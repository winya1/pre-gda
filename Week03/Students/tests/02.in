5
John Smith 5 4 3
John Doe 5 5 4
Jane Doe 4 3 3
John Taxpayer 1 2 3
Joe Shmoe 0 0 0
