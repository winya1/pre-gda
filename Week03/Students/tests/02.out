Case #1: Smith, John: Grades: [5, 4, 3] (avg: 4.0)
Case #2: Doe, John: Grades: [5, 5, 4] (avg: 4.7)
Case #3: Doe, Jane: Grades: [4, 3, 3] (avg: 3.3)
Case #4: Taxpayer, John: Grades: [1, 2, 3] (avg: 2.0)
Case #5: Shmoe, Joe: Grades: [0, 0, 0] (avg: 0.0)
